### PR TITLE
Update banner styles

### PR DESCRIPTION
**Pull request by sftim**
Allow text to be any color, not just white. Leave white text as the default.

/area web-development

### CODE REVIEW

**Comment by sftim** [comment]
Still useful, even though KubeCon has ended. The deploy preview shows an example.

**Comment by sftim** [comment]
Still useful, even though KubeCon has ended. The deploy preview shows an example.

**Comment by reylejano** [comment]
Thank you for the update
/approve

File: assets/scss/_custom.scss [hunk 1]
```diff
@@ -648,6 +648,9 @@ body.td-documentation {
 }
 
 #announcement {
+  // default background is blue; overrides are possible
+  color: #fff;
```
Comment (Sea-n) [inline assets/scss/_custom.scss:652]: nit: Not sure whether `#fff` or `white` is better.

**Review by Sea-n — APPROVED** [review]
wow, thanks!

File: assets/scss/_custom.scss (hunk 1 above)
Comment (zacharysarah) [reply assets/scss/_custom.scss:652]: I like hex codes for the greater degree of specificity they permit. The issue is certainly a conversation we can have as a community. But for now--

/lgtm

**Pull request merged**